### PR TITLE
Update SET doc to match newest rotatiaon process

### PR DESCRIPTION
**Pull request by jwilander**
#### Summary
Changed the intro to be accurate again.

### CODE REVIEW

File: site/content/internal/sustained-engineering.md [hunk 1]
```diff
@@ -9,11 +9,13 @@ The Sustained Engineering Team (SET) is responsible for improving and maintainin
...
+SET is a rotating team that is comprised of engineers from the different feature teams. The rotation is on a two week cycle. Who is currently on SET can be seen in the header of the [~Sustained Engineering](https://community.mattermost.com/core/channels/sustained-engineering) channel.
 
-Each feature team will commit a single engineer to SET for each rotation. While on SET, that engineer should attend their feature team's sprint planning but should not be assigned any work from their feature team. For the month that engineer should primarily be working on issues for SET. If SET has no tickets to work on, then the engineer may pick up some work for their feature team.
+Feature teams will commit engineers to SET for each rotation. With there being 4 engineers plus one lead on SET at any given time. While on SET, that engineer should attend their feature team's sprint planning but should not be assigned any work from their feature team. For the month that engineer should primarily be working on issues for SET. If SET has no tickets to work on, then the engineer may pick up some work for their feature team.
```
Comment (amyblais) [inline site/content/internal/sustained-engineering.md:14]:
```suggestion
Feature teams will commit engineers to SET for each rotation. With there being 4 engineers plus one lead on SET at any given time. While on SET, that engineer should attend their feature team's sprint planning but should not be assigned any work from their feature team. For the two weeks that engineer should primarily be working on issues for SET. If SET has no tickets to work on, then the engineer may pick up some work for their feature team.
```

Comment (amyblais) [inline site/content/internal/sustained-engineering.md:14]:
```suggestion
Feature teams will commit 4 engineers plus one lead for each SET rotation. While on SET, that engineer should attend their feature team's sprint planning but should not be assigned any work from their feature team. For the month that engineer should primarily be working on issues for SET. If SET has no tickets to work on, then the engineer may pick up some work for their feature team.
```

Comment (jwilander) [reply site/content/internal/sustained-engineering.md:14]: Doesn't that make it sound like each team gives up four engineers :P

Comment (sbishel) [reply site/content/internal/sustained-engineering.md]: May-be this? Ha, like I should be word-smithing. :)
"Feature teams will commit a total of 4 engineers plus...."

Comment (jwilander) [reply site/content/internal/sustained-engineering.md]: Works for me :)

**Comment by jwilander** [comment]
Beat me to it @amyblais :P

**Pull request merged**